src/DepthChart2.java

```diff
@@ -0,0 +1,69 @@
+import java.util.*;
+
+/*Convention:-Each key in{@code re
+ * } corresponds  The order of players in the stack for each position represents
+ *  their depth chart ranking:the player
+ * Correpondence: A DepthChart2 represents a mapping from position strings
+ * (e.g., "qb", "wr") to a depth-ordered stack of player names (most recent
+ * addition = starter).
+ */
+public class DepthChart2 extends DepthChartSecondary {
+
+    /**
+     * Representation of this.
+     */
+    private Map<String, Stac<String>> rep;
+
+    /**
+     * Ordered list for iterator.
+     */
+    static final List<String> ORDERED_POSITIONS = List.of("qb", "hb", "fb",
+            "wr", "te", "ot", "og", "c", "dt", "de", "ilb", "olb", "cb", "fs",
+            "ss", "k", "p", "ls");
+
+    @Override
+    public DepthChart2() {
+        this.rep = new HashMap<>();
+        for (String position : ORDERED_POSITIONS) {
+            this.rep.put(position, new Stack<>());
+        }
+    }
+
```

Comment:
I would expect to see clear, newInstance, and transferFrom, if you're correctly extending Standard. You do not need to use the OSU API, but if not, I would expect a Standard interface of your own.